src/components/link/stories/LinkGroup.stories.tsx

```diff
@@ -7,6 +7,7 @@ import notes from './LinkGroup.stories.md';
 
 export const basic = () => (
     <LinkGroup>
+        <button type="button">foo</button>
```

Comment:
intentional?

Reply:
good catch, I was testing if TS complains about it. It doesn't :(